<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppViewerClass</class>
 <widget class="QWidget" name="AppViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AppViewer</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>491</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>351</width>
     <height>265</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayoutEdit">
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelChangeName">
      <property name="text">
       <string>Zmień nazwę:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditImage"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelAddImage">
      <property name="text">
       <string>Dodaj obraz:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="labelAddService">
      <property name="text">
       <string>Dodaj usługę:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="labelSetPrice">
      <property name="text">
       <string>Ustaw cenę:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelAddWorker">
      <property name="text">
       <string>Dodaj pracownika:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelAddPatient">
      <property name="text">
       <string>Dodaj pacjenta:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelAddEquipment">
      <property name="text">
       <string>Dodaj wyposażenie:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pushButtonBrowse">
      <property name="text">
       <string>Przeglądaj</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonSaveName">
      <property name="text">
       <string>Zapisz</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditEquipment"/>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditPatient"/>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditWorker"/>
    </item>
    <item row="9" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditPrice"/>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditService"/>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QPushButton" name="pushButtonAddStuff">
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>520</y>
     <width>493</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayoutButtons">
    <item>
     <widget class="QPushButton" name="pushButtonAddClinic">
      <property name="text">
       <string>Dodaj Przychodnię</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAddNails">
      <property name="text">
       <string>Dodaj Studio Paznokci</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonAddBeauty">
      <property name="text">
       <string>Dodaj Salon Piękności</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="text">
       <string>Usuń</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>600</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Zakończ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDemo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>600</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Demo</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="appviewer.qrc"/>
 </resources>
 <connections/>
</ui>
